hr_job_category/__init__.py

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
in init file you can remove the fist line 

the next time you can use `find . -name "*.py" -exec sed -i "/#.*coding\: /d" {} \;` to help you it's on section [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)